hack/devtools/rp_dev_helper.sh

```diff
@@ -0,0 +1,448 @@
+#!/bin/bash -e
+######## Helper file to run full RP dev either locally or using Azure DevOps Pipelines ########
+
+secret_files=("dev-ca.crt" "dev-client.crt" \
+        "portal-client.pem" "firstparty.pem" "localhost.pem" "arm.pem" \
+        "cluster-mdsd-self-signed.pem" "gwy-mdm-self-signed.pem" "gwy-mdsd-self-signed.pem" "rp-mdm-self-signed.pem" \
+        "rp-mdsd-self-signed.pem" "full_rp_id_rsa" "full_rp_id_rsa.pub" "env" )
+
+verify_downloading_secrets() {
+    # Define the expected directory and file names
+    expected_dir="secrets"
+    all_secret_files=("firstparty.key" "vpn-client.key" "vpn-eastus.ovpn" "localhost.crt" "proxy.crt" \
+        "dev-ca.crt" "proxy-client.pem" "vpn-aks-eastus.ovpn" "portal-client.key" "portal-client.pem" \
+        "proxy-client.key" "vpn-ca-cert.pem" "rp-metrics-int.pem" "dev-client.crt" "rp-logging-int.pem" \
+        "vpn-ca.crt" "shared-cluster.kubeconfig" "vpn-client.pem" "firstparty.pem" "portal-client.crt" \
+        "dev-ca.key" "vpn-aks-australiaeast.ovpn" "vpn-client-cert.pem" "vpn-client-key.pem" "dev-client.pem" \
+        "proxy.key" "firstparty.crt" "vpn-client.crt" "vpn-aks-westeurope.ovpn" "localhost.key" "vpn-ca.pem" \
+        "vpn-australiaeast.ovpn" "cluster-logging-int.pem" "vpn-ca.key" "localhost.pem" "proxy.pem" "dev-client.key" \
+        "proxy_id_rsa.pub" "proxy_id_rsa" "vpn-ca-key.pem" "proxy-client.crt" "vpn-westeurope.ovpn" "dev-ca.pem" \
+        "arm.crt" "arm.key" "arm.pem" "cluster-mdsd.crt" "cluster-mdsd.key" "cluster-mdsd.pem" "cluster-mdsd-self-signed.pem" \
+        "gwy-mdm-self-signed.pem" "gwy-mdsd-self-signed.pem" "rp-mdm-self-signed.pem" "rp-mdsd-self-signed.pem" \
+        "full_rp_id_rsa" "full_rp_id_rsa.pub" "env")
+
+    # Validate that the secrets directory and required files exist
+    [ -d "$expected_dir" ] || { echo "Directory '$expected_dir' has not been created."; exit 1; }
+    #  TODO check if only the below files are required
+    for file in "${secret_files[@]}"; do
+    [ -f "$expected_dir/$file" ] || { echo "File '$file' does not exist inside the directory '$expected_dir'."; exit 1; }
+    done
+}
+
+check_jq_installed() {
+  if ! command -v jq &> /dev/null; then
```

Comment:
Not sure if the function is needed at all as the container is installing it already. It might be helpful for running the function outside the container.